transcom/mymove
# B 20010 int

## [B-20010](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A949820)

## Summary

This backlog adds a new Headquarters Office user role. Further downstream tasks will add privileges meant for the HQ role which will limit pages to read only.

Is there anything you would like reviewers to give additional scrutiny?

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Login to the admin application
2. Create a new Office user
3. Assign the Headquarters role and TOO role to the user and save
![image (2)](https://github.com/transcom/mymove/assets/149000222/c9ed8189-5b41-4247-91cd-69d95c4f57d9)

4. Go to the Office app and sign in as the newly created user
5. Click the change user role and verify you can login with the HQ role
![image (3)](https://github.com/transcom/mymove/assets/149000222/a82392e1-f07e-4189-b475-469ed5b26637)
NOTE: At this point you should be able to do what a TOO can do but this story was to just allow a new login role

6. On the Office App login screen click the Request Account button in the top right corner
7. Fill out the form
8. Verify there is a checkbox for Headquarters role and check it
9. Click Submit and make sure it submits the request
10. Login to the Admin App under the Requested Office Users section find your requested account and approve it
11. Go to the Office App and verify you can login to the app with your new requested user

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
